lib/cassia/response_handlers/get_all_routers_status.rb

```diff
@@ -0,0 +1,28 @@
+module Cassia
+  module ResponseHandlers
+    class GetAllRoutersStatus
+      def initialize(access_controller)
+        @access_controller = access_controller
+      end
+
+      def handle(response)
+        if response.success?
+          handle_success(response)
+        else
+          handle_failure(response)
+        end
+        response.success?
+      end
+
+      private
+
+      def handle_success(response)
```

Comment:
I think this function still needs to be done. We should have a `Router` model, that holds all of our router attributes. Then `AccessController` should have a `routers` attribute that is an array of `Router` models